media-source-respec.html

```diff
@@ -2706,6 +2714,317 @@ <h3>Event Summary</h3>
+              [=ManagedSourceBuffer/memory pressure=] algorithm.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      <h3>Algorithms</h3>
+      <h4>Buffered Change</h4>
+      <p>
+        The following steps are run at the completion of all operations to the {{ManagedSourceBuffer}}
+        |buffer:ManagedSourceBuffer| that would cause a |buffer|'s {{SourceBuffer/buffered}} to change.
+        That is once {{SourceBuffer/appendBuffer()}}, {{SourceBuffer/remove()}} or
+        [=ManagedSourceBuffer/memory pressure=] algorithm have completed.
+      </p>
+      <ol>
+        <li>
+          Let |previous buffered ranges:normalized TimeRanges| equal the
+          {{SourceBuffer/buffered}} attribute before the changes occurred.
+        </li>
+        <li>
+          Let |new buffered ranges:normalized TimeRanges| equal the new
+          {{SourceBuffer/buffered}} {{TimeRanges}}.
+        </li>
+        <li>
+          Let |added:normalized TimeRanges| equal the |previous buffered
+          ranges:normalized TimeRanges| subtracted from |new buffered
+          ranges:normalized TimeRanges|.
+        </li>
+        <li>
+          Let |removed:normalized TimeRanges| equal the |new buffered
+          ranges:normalized TimeRanges| subtracted from |previous buffered
+          ranges:normalized TimeRanges|.
+        </li>
+        <li>
+          Let |eventInitDict| be a new {{BufferedChangeEventInit}} dictionary
+          initialized with |added| as its {{BufferedChangeEventInit/addedRanges}} and
+          |removed| as its {{BufferedChangeEventInit/removedRanges}}
+        </li>
+        <li>
+          [=Queue a task=] to [=fire an event=] named {{bufferedchange}} at |buffer|
+          using the {{BufferedChangeEvent}} interface, initialized with
+          |eventInitDict|.
+        </li>
+      </ol>
+      <h4>
+        <dfn data-dfn-for="ManagedSourceBuffer">Memory pressure</dfn>
+      </h4>
+      <ol>
+        <li>
+          <dl class="switch">
+            <dt>
+              If |managedSourceBuffer| is not in [=this=]'s
+              {{MediaSource/activeSourceBuffers}}:
+            </dt>
+            <dd>
+              <ol>
+                <li>
+                  Run the [=coded frame removal=] algorithm with 0 and positive infinity equal to the removal range start and end timestamp respectively, and abort these steps.
+                </li>
+              </ol>
+            </dd>
+          </dl>
+        </li>
+        <li>
+          Let |removal ranges:normalized TimeRanges| equal a list of presentation
+          time ranges that can be evicted from the presentation to ensure
+          uninterrupted playback from {{HTMLMediaElement.currentTime}} until such
+          presentation could be fetched again.
+          <p class="note">
+            Implementations can use different methods for selecting |removal ranges|
+            so web applications shouldn't depend on a specific behavior. The web
```

Comment:
Use of RFC2119 in non-normative part of a specification (such as "Note:", etc.) is forbidden, so we need to rephrase this.

Reply:
this is existing verbiage already used earlier. So if this is changed it would be better done in a separate PR.